pages/configuration.mdx

```diff
@@ -1 +1,113 @@
+  <Card title="Basic" href="/configuration#basic-configuration" />
+  <Card title="Search" href="/configuration/search" />
+  <Card title="Locale" href="/configuration/locale" />
+  <Card title="Map" href="/configuration/map" />
+  <Card title="Theme" href="/configuration/theme" />
+</Cards>
+
+## Basic Configuration
+
+A custom Canopy configuration file can be created with the following steps:
+
+1. Find your `config/` directory
+2. Copy `canopy.sample.json` to `canopy.json`
+3. Make updates to both the **prod** and **dev** configurations
+
+## Properties
+
+| Property     | Type            | Description                            |
+| ------------ | --------------- | -------------------------------------- |
+| `collection` | `string`        | IIIF Collection URI                    |
+| `label`      | `label `        | IIIF Presentation 3.0 label property   |
+| `summary`    | `summary`       | IIIF Presentation 3.0 summary property |
+| `featured`   | `Array<string>` | Array of IIIF Manifest URIs            |
+| `metadata`   | `Array<string>` | Array of `metadata` labels             |
+
+### collection
+
+The `collection` property is required and must be the id of the referenced source IIIF Collection. Collections of Collections are not currently supported.
+
+```json
+"collection": "https://iiif.bodleian.ox.ac.uk/iiif/collection/hobhouse"
+```
+
+### label _and_ summary
+
+The Canopy IIIF site **title** and **description** are respectively the `label` and `summary` IIIF Presentation API properties of the set IIIF Collection resource. You can optionally override this by providing a valid Presentation 3.0 [label](https://iiif.io/api/presentation/3.0/#label) and/or [summary](https://iiif.io/api/presentation/3.0/#summary) property. These values are transformed to meta elements in the `<head>` of the HTML document.
+
+```json
+"label": { "none": ["Hobhouse"] },
+"summary": { "none": ["Manuscripts from the archive of Emily Hobhouse."] }
+```
+
+### featured
+
+You can inform Canopy IIIF of featured Manifests by providing an array of ids. These must be within the referenced collection resource and the Manifest URIs must be an exact match. These Manifests will render throughout the interface in featured components.
+
+**Warning:** In the current pre-release, featured Manifests must have an Image body on the first Canvas.
+
+```json
+"featured": [
+  "https://iiif.bodleian.ox.ac.uk/iiif/manifest/8da97e8c-4e12-457d-aad8-3327b3aec183.json",
+  "https://iiif.bodleian.ox.ac.uk/iiif/manifest/2968d5c7-3718-44ef-92ea-ee4cc58cc677.json"
+]
+```
+
+### metadata
+
+Curating Metadata allows implementers of Canopy IIIF to select metadata labels that provide use to end users. An optimal case is a label common to all or most manifests with some in diversity of values across those resources. Metadata labels that are curated will be automatically included as featured elements on the homepage, the metadata page, linking from works, and as facets on the search page.
```

Comment:
This sounds a little weird to me. I'm not sure my suggestion is quite right, but I don't like your first sentence and I think the last sentence should come up before the second.

```suggestion
Curating Metadata allows implementers of Canopy IIIF to select metadata fields that provide additional access points to end users. The fields can be specified by listing the labels of each metadata property found in the `metadata` property of the manifest. Metadata labels that are curated will be automatically included as featured elements on the homepage, the metadata page, linking from works, and as facets on the search page. An optimal case is a label common to all or most manifests with some in diversity of values across those resources.
```